nilearn/signal.py

```diff
@@ -374,10 +398,6 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
 
     Filtering is only meaningful on evenly-sampled signals.
 
-    According to Lindquist et al. (2018), removal of confounds will be done
-    orthogonally to temporal filters (low- and/or high-pass filters), if both
-    are specified.
-
```

Comment:
Why was this removed?

Reply:
@kchawla-pi Could you restaure this comment ?